### PR TITLE
Created issue form template

### PR DESCRIPTION
**Pull request by Vidya2606**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issueForm.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        required: false
+   - type: textarea
+     id: what-happened
+     attributes:
+        label: What happened?
+        description: Also tell us, what did you expect to happen?
+        placeholder: Tell us what you see!
+     validations:
+        required: true
+   - type: textarea
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/issueForm.yml:28]: type: input is better

File: .github/ISSUE_TEMPLATE/issueForm.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+     id: version
+     attributes:
+        label: Version
+        description: What action version are you running?
+        placeholder: Mention the action version, runner or any other information
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/issueForm.yml:33]: add another input for runner (self-hosted, operating system)

File: .github/ISSUE_TEMPLATE/issueForm.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+     validations:
+        required: true
+   - type: dropdown
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/issueForm.yml:36]: remove browsers. Browsers aren't relevant to GitHub actions

File: .github/ISSUE_TEMPLATE/issueForm.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+           - Chrome
+           - Safari
+           - Microsoft Edge
+   - type: textarea
+     id: logs
+     attributes:
+        label: Relevant log output
+        description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+        render: shell
+   - type: checkboxes
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/issueForm.yml:52]: you can remove this

File: .github/ISSUE_TEMPLATE/issueForm.yml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        options:
+           - Firefox
+           - Chrome
+           - Safari
+           - Microsoft Edge
+   - type: textarea
+     id: logs
+     attributes:
+        label: Relevant log output
+        description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/issueForm.yml:50]: mention that they should run it in debug mode for the most verbose logs.

Comment (OliverMKing) [reply .github/ISSUE_TEMPLATE/issueForm.yml:50]: mention that they could also attach a screenshot

File: .github/ISSUE_TEMPLATE/issueForm.yml [hunk 6]
```diff
@@ -0,0 +1,59 @@
+name: Bug Report
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/issueForm.yml:1]: We should also have the ability for them to submit a feature request. We should make another form for that requesting

1. Feature information
2. Example functionality
3. Links to relevant docs

**Review by OliverMKing — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/issueForm.yml [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+description: File a bug report
+title: 'Bug: '
+labels: ['bug', 'triage']
+assignees:
+   - Azure/aks-atlanta
```
[the remark this answers is not in the document]
Comment (Vidya2606) [reply .github/ISSUE_TEMPLATE/issueForm.yml:6]: I get this error when I add @ so I removed it --- "SyntaxError: Plain value cannot start with reserved character @"

File: .github/ISSUE_TEMPLATE/featureRequestForm.yml [hunk 8]
```diff
@@ -0,0 +1,39 @@
+name: Feature Request
+description: File a Feature Request form, we will respond to this thread with any questions.
+title: 'Feature Request: '
+labels: ['Feature', 'Functionality']
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/featureRequestForm.yml:4]: remove 'Functionality'

Comment (OliverMKing) [reply .github/ISSUE_TEMPLATE/featureRequestForm.yml:4]: No need for both tags

File: .github/ISSUE_TEMPLATE/featureRequestForm.yml [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+assignees:
+   - azure/aks-atlanta
+body:
+   - type: input
+     id: feature request
+     attributes:
+        label: feature request
+        description: also tell us the example functionality and links to relevant docs
+        placeholder: tell us more!
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/featureRequestForm.yml:13]: remove the placeholder

File: .github/ISSUE_TEMPLATE/featureRequestForm.yml [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+     validations:
+        required: true
+   - type: input
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/featureRequestForm.yml:16]: remove the version. we only support latest version

File: .github/ISSUE_TEMPLATE/featureRequestForm.yml [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+     id: version
+     attributes:
+        label: version
+        description: what action version are you running?
+        placeholder: mention the action version
+     validations:
+        required: true
+   - type: input
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/featureRequestForm.yml:24]: runner doesn't matter either for features (we want to support all). remove

File: .github/ISSUE_TEMPLATE/featureRequestForm.yml [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+        label: runner
+        description: what runner are you using?
+        placeholder: mention the runner(self-hosted, operating system)
+     validations:
+        required: true
+   - type: checkboxes
+     id: terms
+     attributes:
+        label: code of Conduct
+        description: by submitting this issue, you agree to follow our [Code of Conduct]()
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/featureRequestForm.yml:36]: Remove this checkbox

File: .github/ISSUE_TEMPLATE/issueForm.yml [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+     id: logs
+     attributes:
+        label: relevant log output
+        description: run in debug mode for the most verbose logs. Please feel free to attach a screenshot of the logs
+        render: shell
+   - type: checkboxes
+     id: terms
+     attributes:
+        label: code of Conduct
+        description: by submitting this issue, you agree to follow our [Code of Conduct]()
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/issueForm.yml:42]: don't need this checkbox. remove

**Review by OliverMKing — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/issueForm.yml [hunk 14]
```diff
@@ -0,0 +1,45 @@
+name: Bug Report
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/issueForm.yml:1]: rename this file to bugReportForm

File: .github/ISSUE_TEMPLATE/bugReportForm.yml [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+        description: also tell us what did you expect to happen?
+        placeholder: tell us what you see!
+     validations:
+        required: true
+   - type: input
+     id: logs
+     attributes:
+        label: relevant log output
+        description: run in debug mode for the most verbose logs. Please feel free to attach a screenshot of the logs
+        render: shell
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/bugReportForm.yml:20]: we also need to know the runner info (os and whether it was a custom runner). It's only not relevant for the freatureRequestForm.

**Review by OliverMKing — CHANGES_REQUESTED** [review]

**Review by OliverMKing — DISMISSED** [review]

**Review by jaiveerk — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bugReportForm.yml [hunk 16]
```diff
@@ -0,0 +1,28 @@
+name: Bug Report
+description: File a bug report, we will respond to this thread with any questions.
+title: 'Bug: '
+labels: ['bug', 'triage']
+assignees: '@Azure/aks-atlanta'
+body:
+   - type: input
+     id: what-happened
+     attributes:
+        label: what happened?
```
Comment (jaiveerk) [inline .github/ISSUE_TEMPLATE/bugReportForm.yml:10]: nit: can we capitalize the first letter of these?

File: .github/ISSUE_TEMPLATE/bugReportForm.yml [hunk 17]
```diff
@@ -0,0 +1,28 @@
...
+        description: also tell us what did you expect to happen?
```
Comment (jaiveerk) [inline .github/ISSUE_TEMPLATE/bugReportForm.yml:11]: nit: feel like "tell us what you expected to happen" reads better

File: .github/ISSUE_TEMPLATE/featureRequestForm.yml [hunk 18]
```diff
@@ -0,0 +1,13 @@
...
+description: File a Feature Request form, we will respond to this thread with any questions.
+title: 'Feature Request: '
+labels: ['Feature']
+assignees: '@Azure/aks-atlanta'
+body:
+   - type: input
+     id: feature request
+     attributes:
+        label: feature request
+        description: also tell us the example functionality and links to relevant docs
```
Comment (jaiveerk) [inline .github/ISSUE_TEMPLATE/featureRequestForm.yml:11]: nit: "Provide example functionality and links to relevant docs" reads a bit better imo but super nitpicky

**Review by davidgamero — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bugReportForm.yml [hunk 19]
```diff
@@ -0,0 +1,28 @@
...
+description: File a bug report, we will respond to this thread with any questions.
+title: 'Bug: '
+labels: ['bug', 'triage']
+assignees: '@Azure/aks-atlanta'
+body:
+   - type: input
+     id: What-happened
+     attributes:
+        label: What happened?
+        description: Tell us what did you expect to happen?
```
Comment (jaiveerk) [inline .github/ISSUE_TEMPLATE/bugReportForm.yml:11]: super nit: would you mind changing this to "Tell us what you expected to happen"? also, I'm a bit confused - is this section in the bug report supposed to outline what actually happened, what the user expected to happen, or both?

Comment (OliverMKing) [reply .github/ISSUE_TEMPLATE/bugReportForm.yml:11]: It should be both. What happened and how it differs from what was expected. Good point here Jaiveer. That can be clarified

Comment (Vidya2606) [reply .github/ISSUE_TEMPLATE/bugReportForm.yml:11]: how does this sound:  "Tell us what happened and how is it different form the expected?"

File: .github/ISSUE_TEMPLATE/bugReportForm.yml [hunk 20]
```diff
@@ -0,0 +1,28 @@
...
+description: File a bug report, we will respond to this thread with any questions.
+title: 'Bug: '
+labels: ['bug', 'triage']
+assignees: '@Azure/aks-atlanta'
+body:
+   - type: input
+     id: What-happened
+     attributes:
+        label: What happened?
+        description: Tell us what happened and how is it different form the expected?
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/bugReportForm.yml:11]: from

Comment (OliverMKing) [reply .github/ISSUE_TEMPLATE/bugReportForm.yml:11]: typo

File: .github/ISSUE_TEMPLATE/bugReportForm.yml [hunk 21]
```diff
@@ -0,0 +1,28 @@
...
+        description: What runner are you using?
+        placeholder: Mention the runner info (self-hosted, operating system)
+     validations:
+        required: true
+   - type: input
+     id: Logs
+     attributes:
+        label: Relevant log output
+        description: Run in debug mode for the most verbose logs. Please feel free to attach a screenshot of the logs
+        render: shell
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/bugReportForm.yml:28]: make this required

File: .github/ISSUE_TEMPLATE/bugReportForm.yml [hunk 22]
```diff
@@ -0,0 +1,28 @@
+name: Bug Report
+description: File a bug report, we will respond to this thread with any questions.
+title: 'Bug: '
+labels: ['bug', 'triage']
+assignees: '@Azure/aks-atlanta'
+body:
```
Comment (OliverMKing) [inline .github/ISSUE_TEMPLATE/bugReportForm.yml:6]: Specifying which inputs were provided for the action

Comment (OliverMKing) [reply .github/ISSUE_TEMPLATE/bugReportForm.yml:6]: add checkbox having user confirm they were using latest version?

**Review by OliverMKing — CHANGES_REQUESTED** [review]

**Review by OliverMKing — APPROVED** [review]

**Review by jaiveerk — APPROVED** [review]

**Pull request closed without merging**